micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Changelog 
 
+## [1.8.24]  [3-Feb-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)